### PR TITLE
[PH] Add Ability to Specify Contract for Performance Tests

### PR DESCRIPTION
**Pull request by ClaytonCalabrese**
Currently does not alter what transaction generator does, so doesn't meaningfully alter the tests yet.

Dependent on https://github.com/AntelopeIO/leap/pull/594 being merged into PH.

### CODE REVIEW

**Review by oschwaldp-oci — COMMENTED** [review]
Per our discussion today, there were a couple things you were going to consider including in this PR.  Let me know when you'd like me to review again.

File: tests/performance_tests/performance_test_basic.py [hunk 1]
```diff
@@ -248,10 +258,27 @@ def setupWalletAndAccounts(self):
         self.account1PrivKey = self.cluster.accounts[0].activePrivateKey
         self.account2PrivKey = self.cluster.accounts[1].activePrivateKey
 
+    def setupContract(self):
+        print(self.specifiedContract)
```
Comment (ClaytonCalabrese) [inline tests/performance_tests/performance_test_basic.py:262]: print to remove

**Comment by ClaytonCalabrese** [comment]
To show that it is indeed activating the contract, I have activated the "payloadless" contract and the code hash changes from 

`code hash: 3873549763963da8063c43b473dbf1c39ff5cd4bfc39d810c71adc9ee3e7805a`
to
`2d681f279833fa041694f4d61a2643caf2ba461238a3d54c146fd9d2bfe072e6`

File: tests/performance_tests/performance_test.py [hunk 2]
```diff
@@ -508,6 +508,7 @@ def main():
     extraNodeosArgs = ENA(chainPluginArgs=chainPluginArgs, httpPluginArgs=httpPluginArgs, producerPluginArgs=producerPluginArgs, netPluginArgs=netPluginArgs)
     testClusterConfig = PerformanceTestBasic.ClusterConfig(pnodes=args.p, totalNodes=args.n, topo=args.s, genesisPath=args.genesis,
                                                            prodsEnableTraceApi=args.prods_enable_trace_api, extraNodeosArgs=extraNodeosArgs,
+                                                           specifiedContract=PerformanceTestBasic.ClusterConfig.SpecifiedContract(args.account_name, args.owner_public_key, args.active_public_key, args.contract_dir, args.wasm_file, args.abi_file),
```
Comment (oschwaldp-oci) [inline tests/performance_tests/performance_test.py:511]: Do you want to use keyword arguments here as well?

**Review by oschwaldp-oci — APPROVED** [review]

**Pull request merged**